src/pages/coaches/horacio-herrera.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Horacio Herrera
+title: Designer and Developer
+companyName: LeanJS
+companyLink: https://leanjs.com
+gitHub: https://github.com/horacioh
+medium: https://medium.com/@hhg2288
+twitter: https://twitter.com/hhg2288
+linkedIn: https://uk.linkedin.com/in/rhoracioherrera
+blockquote: I love teaching and seeing other developers grow...
+imageDescription: React GraphQL Academy coach Horacio Herrera, smiling with sunglasses on.
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Fhoracio.jpg?alt=media
+tags: []
+---
+
+<ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists